### DISCUSSION TITLE
High MOI vs Low MOI

### DISCUSSION THREAD

**Discussion by jpulecio** [Katsevich-Lab/sceptre#153] [Q&A] [answered]
Hi!
We are analyzing the results of two CRISPRa screens using same gRNA library/cells (one using low MOI and another using high MOI) to detect enhancers associated to genes that are not expressed (determined by previous RNA-seq). SCEPTRE analysis show 29 enhancers found only in the high MOI, 28 enhancers found in both and 32 enhancers found only in the low MOI screen. In this context would you consider as valid all the detected regions, regardless if they were found in the high or low MOI setting, or only the ones that are detected in both conditions?  My understanding is that finding a false positive in a CRISPRa setting, particularly in a condition where the cells don't express the genes of interest, is highly unlikely.

**Comment by ekatsevi** [comment, marked as answer]
I would probably consider your discoveries the be the union of the two sets, with the intersection of the two sets representing "high-confidence" findings. Especially if your calibration checks looked good for both screens, there is no reason to worry about excessive false positive findings. The reason why you might find apparently divergent results from two screens has to do with power. Let's suppose you have about 100 real regulatory relationships, but your power to find each of them is about 0.5. Each screen will find about 50 relationships, and the overlap between the two is expected to be about 25. Of course, strong signals have higher power to be discovered, so are more likely to end up being hits in both screens, but if your power on average is not too high (e.g., 0.5), then you are likely not to find certain hits significant in both screens even if they are real.
